geode-redis/src/main/java/org/apache/geode/redis/internal/executor/string/SetBitExecutor.java

```diff
@@ -81,31 +79,30 @@ public void executeCommand(Command command, ExecutionHandlerContext context) {
       }
       stringCommands.set(key, new ByteArrayWrapper(bytes), null);
       command.setResponse(Coder.getIntegerResponse(context.getByteBufAllocator(), 0));
```

Comment:
I think this setReponse call should have been deleted

Reply:
Doh! Thanks.